conman/routes/models.py

```diff
@@ -82,6 +82,27 @@ def handle(self, request, path):
         # Deal with the request
         return handler.handle(request, path)
 
+    def move_to(self, new_url, *, move_children):
```

Comment:
Not sure about the phrase `move_children`... What do you think, @Ian-Foote and @KevinEtienne? Perhaps this ought to be `descendants`? Is there a better keyword you can think of?